### PR TITLE
Minor Changes on generation dispatch form

### PR DESCRIPTION
**Pull request by ghelloumi**
**Before:**
image

**After:**
image

### CODE REVIEW

**Review by Walid-Sahnoun — COMMENTED** [review]
code ok
test ok
ergonomic proposition: 
disable drag&drop mechanism and icon  when only 1 line exist
change tooltip in buttons from "add row" "delete row" "move up row" "move down row" 
to "add", "delete" , "move up" "move down" 
add tooltip on folder "open" or "choose element"

File: src/components/dialogs/network-modifications/generation-dispatch/generation-dispatch-form.js [hunk 1]
```diff
@@ -55,71 +42,81 @@ const GenerationDispatchForm = ({ currentNode, studyUuid }) => {
...
-            titleId={'FiltersListsSelection'}
-        />
+    const generatorsWithoutOutageField = (
+        <Grid container alignItems="center" spacing={2} direction={'row'}>
+            <Grid item xs={5}>
+                <FieldLabel label={'GeneratorsWithoutOutage'} optional />
+            </Grid>
+            <Grid item xs={4}>
+                <DirectoryItemsInput
+                    name={`${GENERATORS_WITHOUT_OUTAGE}`}
```
Comment (trikimahdi) [inline src/components/dialogs/network-modifications/generation-dispatch/generation-dispatch-form.js:89]: you can remove the  `${}`

**Review by trikimahdi — CHANGES_REQUESTED** [review]

File: src/components/dialogs/network-modifications/generation-dispatch/generation-dispatch-form.js (hunk 1 above)
Comment (ghelloumi) [reply src/components/dialogs/network-modifications/generation-dispatch/generation-dispatch-form.js:89]: done, thank you

**Review by Walid-Sahnoun — APPROVED** [review]
review ok
test ok

**Pull request merged**